### ISSUE TITLE
Use non-deprecated method of declaring screen size

### ISSUE THREAD

**Issue by bazmonk** [bazmonk/digigurdy-baz#64]
The SD1306 header currently requires the user to edit it to select the correct 128x64 screen size.  By default the header uses 128x32.

I should mention this in the install notes... but the header also specifies that this is the deprecated way of doing it.

I want to find the "right" way to do it and do that.

**Comment by bazmonk** [comment]
I'll verify at home, but I'm pretty sure this is the latest version.  The
library is up-to-date according to Teensyduino and it matches the upstream
source:

https://github.com/adafruit/Adafruit_SSD1306/blob/master/Adafruit_SSD1306.h

**Comment by bazmonk** [comment]
Checked: I'm using the latest version, but with that one fix.

I ripped out the library, re-installed it as normal from within teensyduino, and compiled to verify that it gave me the same problem Jm Montes reported (and that I saw when I first got started).  This commit uses the SSD1306 library the "new" way.  I included the SCREEN_HEIGHT and WIDTH variables from an online example--it's been there--but then I copied John's way of initiating the display, which doesn't use it.  So now I'm actually using them :-)

Now it works without modifying the library header at all.

SH1106's library (for the blue OLEDs) is a not-"standard" wohno-maker version that nicely works exactly the same way as SSD1306 on purpose.  It has a similar method of uncommenting the right resolution in the header file, but fortunately its default is what we want here.
